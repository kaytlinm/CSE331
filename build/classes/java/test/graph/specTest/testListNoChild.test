#List No Child Test File
#Testst that a graph can be created and a single node with no children can be added and prints no children

CreateGraph graph1
AddNode graph1 node1
ListChildren graph1 node1