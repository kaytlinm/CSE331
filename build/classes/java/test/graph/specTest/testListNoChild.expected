#List No Child Test File
#Testst that a graph can be created and a single node with no children can be added and prints no children

created graph graph1
added node node1 to graph1
the children of node1 in graph1 are: